public/index.pug

```diff
@@ -56,6 +79,10 @@ html(lang="en-US")
           a(class="header__list-icon" title="Legal Disclaimer" href="https://github.com/simple-icons/simple-icons/blob/master/DISCLAIMER.md")
             svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
               path(d="m23.9 9.7-3.54-7.89-3.2e-4 -6.41e-4 -0.0046-0.0102c-0.0119-0.0264-0.026-0.0512-0.0415-0.0748-0.0042-0.0064-0.0092-0.0122-0.0136-0.0183-0.014-0.0194-0.029-0.0377-0.0453-0.0549-0.0064-0.0067-0.0129-0.0134-0.0196-0.0198-0.0179-0.017-0.0368-0.0327-0.0569-0.047-0.0052-0.0038-0.01-0.0081-0.0155-0.0118-0.0252-0.0167-0.0519-0.031-0.0797-0.0432-0.0079-0.0035-0.0161-0.0061-0.0242-0.0092-0.0218-0.0083-0.0439-0.0153-0.0668-0.0207-0.0094-0.0022-0.0187-0.0045-0.0282-0.0062-0.0313-0.0058-0.0631-0.0096-0.0957-0.0096h-6.85c-1.02-1.52-1.02-1.54-2 1.06e-4h-6.86c-0.0326 0-0.0643 4e-3 -0.0957 0.0096-0.00961 0.0017-0.0188 4e-3 -0.0282 0.0062-0.0229 0.0054-0.0451 0.0124-0.0667 0.0207-0.00812 0.0031-0.0163 0.0057-0.0242 0.0092-0.0278 0.0123-0.0545 0.0265-0.0797 0.0432-0.00545 0.0036-0.0103 0.0079-0.0155 0.0118-0.0201 0.0143-0.039 0.0299-0.0569 0.047-0.00684 0.0064-0.0132 0.013-0.0198 0.0198-0.0163 0.0172-0.0313 0.0356-0.0453 0.0549-0.00449 0.0062-0.0094 0.012-0.0136 0.0183-0.0156 0.0236-0.0297 0.0484-0.0415 0.0748l-0.00459 0.0102-3.2e-4 6.4e-4 -3.55 7.91c-0.06 0.0864-0.0958 0.191-0.0958 0.304 0 2.28 1.86 4.14 4.14 4.14s4.14-1.86 4.14-4.14c0-0.113-0.0357-0.218-0.0958-0.304l-3.25-6.37 6.07-0.0229v18.2c-2.55 0.294-7.01 0.381-7 2.5h16c-0.0011-2.03-4.48-2.27-7-2.5v-18.1l5.69-0.0206-2.92 6.49c-8.55e-4 0.0019-0.0013 0.0038-0.0021 0.0058-0.0025 0.0058-0.0043 0.0118-0.0066 0.0176-0.0096 0.0245-0.0173 0.0494-0.0232 0.0746-0.0015 0.0066-0.0034 0.013-0.0047 0.0196-0.0059 0.0306-0.0089 0.0615-0.0093 0.0925 0 0.0029-8.54e-4 0.0054-8.54e-4 0.0083 0 2.28 1.86 4.14 4.14 4.14 2.28 0 4.14-1.86 4.14-4.14 0-0.126-0.0457-0.241-0.119-0.332z")
+        li(class="header__list-item")
+          a(class="header__list-icon popup-trigger" title="Third party extensions" href="#")
```

Comment:
I've added role="button" according to what's said here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

It also says that the <a> tags with role="button" should handle keydown and the Space key, but that's beyond this PR since we would have to do that for all the <a role="button"> tags, not just this one.

Reply:
> PDF and SVG downloads are , though. They behave as buttons due to a class

As far as I know downloads in HTML should be `<a>`, if that's not the case we should consider changing the element we use for the downloads.



> I've added role="button" according to what's said here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
> 
> It also says that the tags with role="button" should handle keydown and the Space key, but that's beyond this PR since we would have to do that for all the tags, not just this one.

`role="button"` should be used when you can't use an actual button.

Why can't we use a `<button>` here? What's the motivation for using `<a>`? Is it [still](https://github.com/simple-icons/simple-icons-website/pull/127#discussion_r735145971) just consistency? If that is more important to you than semantically correct HTML than feel free to leave it as is, but please don't resolve this conversation to keep it open for visibility. (we can agree to disagree, but that doesn't mean to conversation is resolved).